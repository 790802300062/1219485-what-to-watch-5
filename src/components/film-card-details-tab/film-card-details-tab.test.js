import React from "react";
import renderer from "react-test-renderer";
import {filmListMock} from "../../test-data/test-data";
import FilmCardDetailsTab from "./film-card-details-tab";

describe(`FilmCardDetailsTab render`, () => {
  it(`renders FilmCardDetailsTab component with the first mock movie data`, () => {
    const tree = renderer
      .create(
          <FilmCardDetailsTab
            film={filmListMock[0]}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`renders FilmCardDetailsTab component with another movie data`, () => {
    const tree = renderer
      .create(
          <FilmCardDetailsTab
            film={filmListMock[2]}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
