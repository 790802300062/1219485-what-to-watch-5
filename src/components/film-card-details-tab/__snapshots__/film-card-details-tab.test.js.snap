// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCardDetailsTab render renders FilmCardDetailsTab component with another movie data 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Jeff Nichols
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Michael Shannon
         
        <br />
        Joel Edgerton
         
        <br />
        Kirsten Dunst 
         
        
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        1h 52m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Action
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2016
      </span>
    </p>
  </div>
</div>
`;

exports[`FilmCardDetailsTab render renders FilmCardDetailsTab component with the first mock movie data 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Nicolas Winding Refn
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Tom Hardy
         
        <br />
        Kelly Adams
         
        <br />
        Luing Andrews
         
        
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        1h 32m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Action
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2008
      </span>
    </p>
  </div>
</div>
`;
